ingestion/src/metadata/ingestion/source/database/mysql/connection.py

```diff
@@ -38,6 +42,14 @@ def get_connection(connection: MysqlConnection) -> Engine:
     """
     Create connection
     """
+    if hasattr(connection.authType, "azureConfig"):
+        azure_client = AzureClient(connection.authType.azureConfig).create_client()
+        if not connection.authType.azureConfig.scopes:
+            raise ValueError("Azure Scopes are not provided")
```

Comment:
if they are required, should we mark them as required in the JSON Schema? 🤔 I might be missing something

Reply:
and I guess we'll followup this PR with the docs right

Reply:
Scopes are just needed for Postgresql and Mysql Azure Auth, not at places like Datalake / ADLS